lightning_client.go

```diff
@@ -3469,3 +3474,30 @@ func (s *lightningClient) QueryRoutes(ctx context.Context,
 		TotalAmtMsat:  lnwire.MilliSatoshi(route.TotalAmtMsat),
 	}, nil
 }
+
+func (s *lightningClient) CheckMacaroonPermissions(ctx context.Context,
+	macaroon []byte, permissions []MacaroonPermission) (bool, error) {
+
+	rpcPermissions := make([]*lnrpc.MacaroonPermission, 0)
```

Comment:
nit: remove the `0` argument.

Reply:
I think that argument is required unless i'm missing something, would len(permissions) be better?